--- a/package/base-files/files/bin/config_generate
+++ b/package/base-files/files/bin/config_generate
@@ -297,10 +297,10 @@ generate_static_system() {
set system.ntp='timeserver'
		set system.ntp.enabled='1'
		set system.ntp.enable_server='0'
-		add_list system.ntp.server='ntp1.aliyun.com'
-		add_list system.ntp.server='time1.cloud.tencent.com'
-		add_list system.ntp.server='time.ustc.edu.cn'
-		add_list system.ntp.server='cn.pool.ntp.org'
+		add_list system.ntp.server='time.cloudflare.com'
+		add_list system.ntp.server='time.stdtime.gov.tw'
+		add_list system.ntp.server='time.google.com'
+		add_list system.ntp.server='clock.stdtime.gov.tw'
	EOF

	if json_is_a system object; then
